ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -187,98 +187,6 @@ function run_command_oc() {
     echo "${ret_val}"
 }
 
-function check_clusteroperators() {
-    local tmp_ret=0 tmp_clusteroperator input column last_column_name tmp_clusteroperator_1 rc null_version unavailable_operator degraded_operator
-
-    echo "Make sure every operator do not report empty column"
-    tmp_clusteroperator=$(mktemp /tmp/health_check-script.XXXXXX)
-    input="${tmp_clusteroperator}"
-    ${OC} get clusteroperator >"${tmp_clusteroperator}"
-    column=$(head -n 1 "${tmp_clusteroperator}" | awk '{print NF}')
-    last_column_name=$(head -n 1 "${tmp_clusteroperator}" | awk '{print $NF}')
-    if [[ ${last_column_name} == "MESSAGE" ]]; then
-        (( column -= 1 ))
-        tmp_clusteroperator_1=$(mktemp /tmp/health_check-script.XXXXXX)
-        awk -v end=${column} '{for(i=1;i<=end;i++) printf $i"\t"; print ""}' "${tmp_clusteroperator}" > "${tmp_clusteroperator_1}"
-        input="${tmp_clusteroperator_1}"
-    fi
-
-    while IFS= read -r line
-    do
-        rc=$(echo "${line}" | awk '{print NF}')
-        if (( rc != column )); then
-            echo >&2 "The following line have empty column"
-            echo >&2 "${line}"
-            (( tmp_ret += 1 ))
-        fi
-    done < "${input}"
-    rm -f "${tmp_clusteroperator}"
-
-    echo "Make sure every operator column reports version"
-    if null_version=$(${OC} get clusteroperator -o json | jq '.items[] | select(.status.versions == null) | .metadata.name') && [[ ${null_version} != "" ]]; then
-        echo >&2 "Null Version: ${null_version}"
-        (( tmp_ret += 1 ))
-    fi
-
-    echo "Make sure every operator reports correct version"
-    if incorrect_version=$(${OC} get clusteroperator --no-headers | awk -v var="${TARGET_VERSION}" '$2 != var') && [[ ${incorrect_version} != "" ]]; then
-        echo >&2 "Incorrect CO Version: ${incorrect_version}"
-        (( tmp_ret += 1 ))
-    fi
```

Comment:
This block checks co doesn't have empty output and reports correct version. We need to keep them.